include/fty_common_macros.h

```diff
@@ -0,0 +1,35 @@
+/*  =========================================================================
+    fty_common_macros - System-wide preprocessor macros
+
+    Copyright (C) 2014 - 2018 Eaton
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+    =========================================================================
+*/
+
+#ifndef FTY_COMMON_MACROS_H_INCLUDED
+#define FTY_COMMON_MACROS_H_INCLUDED
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define TRANSLATE_ME(...) __VA_ARGS__
```

Comment:
@jana-rapava that will fail later, since you just define the macro as returning its param (i.e. not just the strings, but its accompanying variables). It will only work when encapsulated in a printf like function, but any reference for ex. a variable definition, will abort.
Ex:
* these will work:
char *error = TRANSLATE_ME("fmt");
printf(TRANSLATE_ME("fmt %s", string);
* these will fail
char *error = TRANSLATE_ME("fmt %s", string);

I'd be in favor of defining it empty for now

Reply:
I tlooks like we don't need failing case anywhere in the codebase. Does it make sense to make a macro empty even though it will soon be replaced by another function?